exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/join/HashJoinProbeTemplate.java

```diff
@@ -136,7 +136,9 @@ public void executeProbePhase() throws SchemaChangeException {
           case OK_NEW_SCHEMA:
             if (probeBatch.getSchema().equals(probeSchema)) {
               doSetup(outgoingJoinBatch.getContext(), buildBatch, probeBatch, outgoingJoinBatch);
-              hashTable.updateBatches();
+              if (hashTable != null) {
+                hashTable.updateBatches();
+              }
```

Comment:
Fine.

Reply:
@Ben-Zvi. Thanks Boaz for the quick review. I made the required changes and had a clean run precommit tests.

Just to keep track of these failures. Here are the clean runs on the current commit.
PASS (5.431 s) /root/drillAutomation/framework-master/framework/resources/Functional/schema_change_empty_batch/maprdb/binary_maprdb/emptyMaprDBLeftJoin.sql (connection: 1289554899) (queryID: 25997883-c4c3-afee-bd18-4907353109cd)
PASS (6.403 s) /root/drillAutomation/framework-master/framework/resources/Functional/schema_change_empty_batch/hbase/emptyHbaseLeftJoin.sql (connection: 1037784189) (queryID: 2599792c-e9c0-304e-a82e-ff9438ed3f5b)
PASS (5.392 s) /root/drillAutomation/framework-master/framework/resources/Functional/schema_change_empty_batch/hbase/emptyHbaseRightJoin.sql (connection: 1697676429) (queryID: 259979de-0f7a-f69e-9fcb-3b70f8eca3df)
PASS (5.515 s) /root/drillAutomation/framework-master/framework/resources/Functional/schema_change_empty_batch/maprdb/binary_hbase/emptyMaprDBRightJoin.sql (connection: 1143786034) (queryID: 2599790d-5a7f-d737-a3dd-83baf15dcf94)